content/en/security/suppressions.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Suppressions
+kind: documentation
+disable_toc: false
+further_reading:
+- link: "security/detection_rules/"
+  tag: "Documentation"
+  text: "Learn more about detection rules"
+---
+
+<div class="alert alert-warning">Suppressions are available for Cloud Security Management Threats and Cloud SIEM.</div>
+
+<div class="alert alert-warning">Suppression rules are replacing suppression queries in detection rules. The legacy suppression queries will be deprecated on April 1, 2024. See <a href="#migrate-legacy-suppression-queries-to-suppression-rules">Migrate legacy suppression queries to suppression rules</a> for more information. </div>
+
+## Overview
+
+Suppressions are specific conditions for when a signal should not be generated, which can improve the accuracy and relevance of the signals that are generated.
+
+## Suppression routes
+
+You can set up suppressions within individual [detection rules](#detection-rules), or define a separate [suppression rule](#suppression-rules) to suppress signals more broadly across multiple detection rules.
+
+### Detection rules
+
```

Comment:
You might move the warning to here, since the user won't have much context for understanding the warning if it's at the top.

Reply:
Good idea! done.